examples/multi_agent/MANet.py

```diff
@@ -0,0 +1,29 @@
+import torch
+import numpy as np
+from torch import nn
+
+from tianshou.data import to_torch
+
+
+class Net(nn.Module):
+    def __init__(self, layer_num, state_shape, action_shape=0, device='cpu',
+                 softmax=False):
+        super().__init__()
+        self.device = device
+        self.model = [
+            nn.Linear(np.prod(state_shape), 128),
+            nn.ReLU(inplace=True)]
+        for i in range(layer_num):
+            self.model += [nn.Linear(128, 128), nn.ReLU(inplace=True)]
+        if action_shape:
+            self.model += [nn.Linear(128, np.prod(action_shape))]
+        if softmax:
+            self.model += [nn.Softmax(dim=-1)]
+        self.model = nn.Sequential(*self.model)
+
+    def forward(self, s, state=None, info={}):
+        s = to_torch(s, device=self.device, dtype=torch.float)
+        batch = s.shape[0]
+        s = s.view(batch, -1)
```

Comment:
`unsqueeze(-1)`

Reply:
The code is copied from other examples. Maybe this can be fixed by another PR.

Reply:
``unsqueeze(-1)`` is not the same as ``view(batch, -1)``.
```
a = torch.ones(4, 5, 6)
b = a.view(a.shape[0], -1)
c = a.unsqueeze(-1)
a.shape
Out[58]: torch.Size([4, 5, 6])
b.shape
Out[59]: torch.Size([4, 30])
c.shape
Out[60]: torch.Size([4, 5, 6, 1])
```

Reply:
I find that ``a.flatten(1)`` does the trikc.

Reply:
Indeed I misread and was about to suggest `flatten` :+1:

Reply:
Fixed in ba7c184b83bbdc9ac960aaab99b9835f78d1c42e . This file will be removed.